request.go

```diff
@@ -471,20 +491,29 @@ func unmarshalNode(data *Node, model reflect.Value, included *map[string]*Node)
 					so unmarshal and set fieldValue only if data obj is not null
 				*/
 				if relationship.Data == nil {
+
+					// Explicit null supplied for the field value
+					// If a nullable relationship we set the
+					if isExplicitNull && strings.HasPrefix(fieldType.Type.Name(), "NullableRelationship[") {
```

Comment:
Would it be enough to simply check `isExplicitNull`? It looks like the only way it can be true here is if we have a `NullableRelationship`.